<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>525</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>570</width>
    <height>525</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>570</width>
    <height>525</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ThunderBug</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="BugSelectGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>551</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Select a bug report section</string>
    </property>
    <widget class="QComboBox" name="BugSelectButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>531</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>General gameplay and balance</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Client and launcher crashes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Graphic and devices</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>UI, text and localisation</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Game controls and devices</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Map, missions and AI</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Flight and damage model</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sound and devices</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Connection and update</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="GenerateButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>480</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate report</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>551</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_game_modetype">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <attribute name="title">
      <string>Game mode/type</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Difficulty</string>
      </property>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>arcade</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>realistic</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>simulator</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>261</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>game mode</string>
      </property>
      <widget class="QRadioButton" name="radioButton_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>plane</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>mixed tanks</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>mixed ships</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_7">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>event</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_8">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>other</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="customgametype">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>60</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_description">
     <attribute name="title">
      <string>Description</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>521</width>
        <height>311</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Description :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>521</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Warning : without a proper description, your bug report will be dismissed</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_replays">
     <attribute name="title">
      <string>Replays</string>
     </attribute>
     <widget class="QListView" name="availableReplay_listView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>521</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
     <widget class="QListView" name="addedReplay_listView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>201</y>
        <width>521</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addReplayButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>add replay</string>
      </property>
     </widget>
     <widget class="QPushButton" name="removeReplayButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>170</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>remove replay</string>
      </property>
     </widget>
     <widget class="QPushButton" name="scanReplayButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>170</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Scan for replay</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="filterReplaycheckBox">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>170</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Filter by selected game mode/type</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_clientlogs">
     <attribute name="title">
      <string>Client logs</string>
     </attribute>
     <widget class="QListView" name="availableGameLog_listView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>521</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addgameLogButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>add game log</string>
      </property>
     </widget>
     <widget class="QPushButton" name="removeGameLogButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>170</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>remove game log</string>
      </property>
     </widget>
     <widget class="QPushButton" name="scanGameLogButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>170</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Scan for logs</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addGameLogfromReplayButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>170</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>add log of selected replay</string>
      </property>
     </widget>
     <widget class="QListView" name="addedGameLog_listView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>521</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_launcherlogs">
     <attribute name="title">
      <string>Launcher logs</string>
     </attribute>
     <widget class="QListView" name="availableLauncherLog_listView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>521</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addLauncherLogButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>add log</string>
      </property>
     </widget>
     <widget class="QPushButton" name="removLauncherLogButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>170</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>remove log</string>
      </property>
     </widget>
     <widget class="QPushButton" name="scanLauncherLogButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>170</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Scan for logs</string>
      </property>
     </widget>
     <widget class="QListView" name="addedLauncherLog_listView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>521</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_screenshots">
     <attribute name="title">
      <string>Screenshots</string>
     </attribute>
     <widget class="QPushButton" name="removeScreenshotButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>340</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>remove</string>
      </property>
     </widget>
     <widget class="QListView" name="screenchotslistView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>521</width>
        <height>321</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="addScreenshotButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>add</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_dxdiag">
     <attribute name="title">
      <string>DxDiag</string>
     </attribute>
     <widget class="QLabel" name="dxdiaginfo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>521</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>generate DxDiag</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>select output directory</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
